packages/backend/src/services/video.ts

```diff
@@ -0,0 +1,81 @@
+import { VideoModel, VideoDoc, VideoInterface } from '@model/video'
+import { Request, Response } from 'express'
+
+/**
+ * Get all videos
+ * @returns array of video document
+ */
+export const getAllVideos = async (
+  lastId: string,
+  limit: number
+): Promise<VideoDoc[]> => {
+  if (lastId === 'NULL') {
+    return await VideoModel.find().limit(limit)
```

Comment:
this is a `redundant async` i think
```suggestion
    return VideoModel.find().limit(limit).exec()
```